imports/plugins/core/orders/client/components/orderSummary.js

```diff
@@ -1,8 +1,35 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 import moment from "moment";
+import { Reaction } from "/client/api";
 import { Badge, ClickToCopy } from "@reactioncommerce/reaction-ui";
 
+/**
+ * getBillingInfo
+ *
+ * @summary get proper billing object as per current active shop
+ * @param {Object} order - order object to check against
+ * @return {Object} proper billing object to use
+ */
```

Comment:
a few notes on this function and use of `find`
- While it's perfectly valid javascript, as an organization, Reaction is moving towards enforcing the use of parens around your function args.  e.g. `billing.find(billing => x === y)` is not as clear as `billing.find((billing) => x === y)`
-  If you have to break your `find` into multiple lines, probably better for readability to use curly brackets and a `return`
- If you're falling back to an empty object, it's more readable if you put the result of the `find` into a variable. I didn't immediately realize what the ` || {}` was part of the return initially.

Example with all suggestions.
```
function getBillingInfo(order) {
  const billingInfo = order.billing.find((billing) => {
    return billing.shopId === Reaction.getShopId();
  }
  return billingInfo || {};
}
```

Reply:
After chatting with @jshimko another pattern that we're using in React if you don't want the curly brackets or return statement
```
order.billing.find((billing) => (
  billing.shopId === Reaction.getShopId()
)
```
I'm not 100% I love this outside of the React context and still prefer to either keep a single line or use curlies and a `return`, but it would be better .